test_action.py

```diff
@@ -23,6 +24,23 @@
 MOCK_TIME = datetime.utcnow()
 
 
+class TestUtils(TestCase):
+    def test_check_aws_credentials(self):
+        with patch.dict(os.environ, {"AWS_ACCESS_KEY_ID": "fake", "AWS_SECRET_ACCESS_KEY": "fake"}):
+            action.check_aws_credentials()
+
+        with self.assertRaises(ValueError):
+            action.check_aws_credentials()
+
+    def test_get_region(self):
+        for variable in ("AWS_REGION", "AWS_DEFAULT_REGION"):
+            with patch.dict(os.environ, {variable: "eu-central-1"}):
+                action.get_region()
```

Comment:
Maybe use an assertEquals here so it actually tests something?

Reply:
Ha-ha, good point, fixed - thanks!